### PR TITLE
Feature  Create Plant Journal Entry

### PR DESCRIPTION
**Pull request by MartaJaekel**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ const nextConfig = {
     styledComponents: true,
   },
   reactStrictMode: true,
+  images: {
+    unoptimized: true,
+  },
```
Comment (doemser) [inline next.config.js:9]: why is this necessary? wouldnt it be better to have optimized images?

**Review by doemser — CHANGES_REQUESTED** [review]
looks good, just some minor things

File: components/JournalForm/index.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+  const [description, setDescription] = useState("");
+  const [careTipps, setCareTipps] = useState("");
+  const [location, setLocation] = useState("");
+
+  const router = useRouter();
+  const goBack = () => {
+    router.back();
+  };
+  const isUrlValid = (url) => {
+    const urlRegex = /^(https?:\/\/)?([\w-]+\.)+[\w-]+(\/[\w-./?%&=]*)?$/;
```
[the remark this answers is not in the document]
Comment (MartaJaekel) [reply components/JournalForm/index.js:20]: @doemser you are right , i replaced it with the pattern attribute in the HTML, is more understandable and easy to implement  :)

File: components/JournalEntryCard/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  max-width: 300px;
+`;
+
+const StyledEntryImage = styled.img`
+  margin-right: 16px;
+`;
+
+const StyledEntryInfo = styled.p`
+  flex: 1;
+`;
```
Comment (mbosselmann) [inline components/JournalEntryCard/index.js:37]: You could work with the figure element. :) `StyledEntryCard` would be the `figure` and `StyledEntryInfo` can be the `figcaption`. :)

File: pages/journal/[id].js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  gap: 1rem; 
+  align-items: center; 
+  border-bottom: 2px solid var(--color-grey);
+  padding: 1rem 0;
+`;
+
+const StyledSection = styled.section`
+  flex: 1;
+  overflow: hidden;
+`;
```
Comment (mbosselmann) [inline pages/journal/[id].js:85]: These stylings could also apply to the `StyledArticle` or? If yes you do not need `StyledSection` or? 🤔

File: pages/journal/[id].js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+const StyledDescription = styled.h3`
```
Comment (mbosselmann) [inline pages/journal/[id].js:87]: After setting a `h1` element the next heading should be an `h2` [(see MDN)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes)

File: pages/journal/entry.js [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+          <StyledInput
+            type="url"
+            id="url"
+            name="url"
+            placeholder="Image Upload URL"
+            pattern="https://.*"
+            onChange={(event) => setUrl(event.target.value)}
+            required
+          />
+          <StyledLabel htmlFor="name"></StyledLabel>
```
Comment (mbosselmann) [inline pages/journal/entry.js:55]: Even though the labels are hidden, there should be some text inside. :)

File: pages/journal/entry.js [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+            placeholder="Location"
+            onChange={(event) => setLocation(event.target.value)}
+          />
+
+          <StyledButtonContainer>
+            <StyledButton type="reset">Cancel</StyledButton>
+            <StyledButton type="submit">Save</StyledButton>
+          </StyledButtonContainer>
+        </StyledForm>
+      </main>{" "}
```
Comment (mbosselmann) [inline pages/journal/entry.js:94]: The white space is not needed or? :)

File: pages/journal/index.js [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+import { useRouter } from "next/router";
+import Image from "next/image";
+import EntryCard from "@/components/JournalEntryCard";
+
+export default function JournalOverviewPage({ entries }) {
+  const router = useRouter();
+  const goBack = () => {
+    router.back();
+  };
+  console.log(entries);
```
Comment (mbosselmann) [inline pages/journal/index.js:14]: One console.log that is not needed anymore. :)

File: pages/journal/index.js [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+  color: white;
+  font-weight: 600;
+  border-radius: 8px;
+  padding: 0.6rem 0.4rem;
+  cursor: pointer;
+  background-color: var(--color-green);
+`;
+const StyledLink = styled.a`
+  text-decoration: none;
+`;
```
Comment (mbosselmann) [inline pages/journal/index.js:76]: We always want to work with the next/link component. :)

File: pages/journal/index.js [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+                  </StyledLink>
+                </li>
+              ))
+            ) : (
+              <StyledParagraph>
+                You don&apos;t have any entries yet
+                <br />
+                start with your first entry.
+              </StyledParagraph>
+            )}{" "}
```
Comment (mbosselmann) [inline pages/journal/index.js:48]: Another not needed white space or? :)

File: pages/journal/index.js [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+  background-color: var(--color-green);
+  border-radius: 50%;
+  width: 30px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+  border: none;
+  z-index: 2;
+`;
+const StyledEntriesContainer = styled.div`
```
Comment (mbosselmann) [inline pages/journal/index.js:91]: Instead of creating an extra `div` container you can also style the `ul` directly. :)

**Review by mbosselmann — COMMENTED** [review]
Hi Team PlantBuddy,
good job! 👍 Left some remarks for you to consider. :)

**Review by mbosselmann — COMMENTED** [review]
LGTM! 🦊

**Pull request merged**